yarn.lock

```diff
@@ -263,6 +263,19 @@
     path-browserify "^1.0.0"
     url-parse "~1.5.1"
 
+"@jupyterlab/coreutils@^5.1.13":
```

Comment:
The `yarn.lock` should not need changes. Would you mind restoring the `master` version and check it does not get updated with `@jupyterlab/translation` added as dependency?